### PR TITLE
feat(mobile): add service navigation to mobile

### PR DESCRIPTION
**Pull request by Kiya971**
We can now access the service page through the header bar in the Area Page

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -68,29 +68,29 @@ services:
...
-      - client_mobile
-    environment:
-      REACT_APP_SERVER_URL: "${SERVER_IP}:${SERVER_PORT}"
-      REACT_APP_GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
-    volumes:
-      - ./web/src:/app/src
     ports:
       - 8081:8081
 
+  # client_web:
```
Comment (Benjicatch) [inline docker-compose.dev.yml:74]: to decomment

File: mobile/Dockerfile.dev [hunk 2]
```diff
@@ -10,6 +10,6 @@ RUN npm install
 
 COPY . .
 
-EXPOSE 8083
+EXPOSE 8081
```
Comment (Benjicatch) [inline mobile/Dockerfile.dev:13]: reput 8083

**Review by Benjicatch — CHANGES_REQUESTED** [review]

File: mobile/src/pages/Service.tsx [hunk 3]
```diff
@@ -1,88 +1,176 @@
...
+  // headerText: {
+  //   textDecorationColor: '#2B2277',
+  //   fontSize: 30,
+  // },
+  // bottomBar: {
+  //   backgroundColor: 'rgba(197, 192, 255, 1)',
+  //   width: '100%',
+  //   position: 'absolute',
+  //   bottom: 0,
+  // },
```
Comment (Miou-zora) [inline mobile/src/pages/Service.tsx:173]:
```suggestion
```

File: mobile/src/pages/Service.tsx [hunk 4]
```diff
@@ -1,88 +1,176 @@
...
+    }
+  }, [listServices, token]);
+
   return (
     <SafeAreaView style={backgroundStyle}>
-      <StatusBar
+      {/* <StatusBar
         barStyle={'light-content'}
         backgroundColor={backgroundStyle.backgroundColor}
+      /> */}
```
Comment (Miou-zora) [inline mobile/src/pages/Service.tsx:73]:
```suggestion
```

File: mobile/src/pages/Service.tsx [hunk 5]
```diff
@@ -1,88 +1,176 @@
...
+        <ServiceCard
+          serviceName={'Spotify'}
+          status={stateSpotify}
+          image={Spotify}
+          userName={''}
+          link={'http://10.0.2.2:8080/auth/spotify'}
+        />
       </View>
-      <Appbar style={styles.bottomBar}>
+      {/* <Appbar style={styles.bottomBar}>
```
Comment (Miou-zora) [inline mobile/src/pages/Service.tsx:115]: Remove this useless comment

File: mobile/src/pages/Service.tsx [hunk 6]
```diff
@@ -1,88 +1,176 @@
...
+        ]}
       />
-      <View style={styles.listContainer}>
-        <ServiceList />
+      <View style={styles.servicesCardArea}>
+        <ServiceCard
+          serviceName={'Google'}
+          status={stateGoogle}
+          image={Google}
+          userName={''}
```
Comment (Miou-zora) [inline mobile/src/pages/Service.tsx:90]: Do you really need this broken feature ?

File: mobile/src/pages/Register.tsx [hunk 7]
```diff
@@ -29,12 +30,26 @@ const Register = ({navigation}) => {
...
       })
       .catch(error => {
-        console.error(error);
+        if (error.response.status === 403) {
+          setMessage('Mail already exist');
+        } else if (error.response.status === 400) {
+          setMessage('Invalid Mail or Password');
+        } else {
+          setMessage('Error');
+          console.error(error.value);
```
Comment (Miou-zora) [inline mobile/src/pages/Register.tsx:50]:
```suggestion
```

**Review by Miou-zora — CHANGES_REQUESTED** [review]

**Comment by Miou-zora** [comment]
image

File: mobile/src/pages/Service.tsx (hunk 6 above)
Comment (AlexGuillard) [reply mobile/src/pages/Service.tsx:86]: Yes

File: mobile/src/pages/Service.tsx (hunk 5 above)
Comment (AlexGuillard) [reply mobile/src/pages/Service.tsx]: its romved

**Review by Miou-zora — APPROVED** [review]

**Review by Benjicatch — APPROVED** [review]

**Pull request merged**